homeassistant/components/fritzbox/sensor.py

```diff
@@ -87,6 +82,26 @@ class FritzSensorEntityDescription(
         suitable=lambda device: device.has_powermeter,  # type: ignore[no-any-return]
         native_value=lambda device: device.power / 1000 if device.power else 0.0,
     ),
+    FritzSensorEntityDescription(
+        key="electric_potential",
+        name="Electric Potential",
+        native_unit_of_measurement=ELECTRIC_POTENTIAL_VOLT,
+        device_class=SensorDeviceClass.VOLTAGE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suitable=lambda device: device.has_powermeter,  # type: ignore[no-any-return]
+        native_value=lambda device: device.voltage if device.voltage else 0.0,
+    ),
+    FritzSensorEntityDescription(
+        key="electric_current",
+        name="Electric Current",
+        native_unit_of_measurement=ELECTRIC_CURRENT_AMPERE,
+        device_class=SensorDeviceClass.CURRENT,
+        state_class=SensorStateClass.MEASUREMENT,
+        suitable=lambda device: device.has_powermeter,  # type: ignore[no-any-return]
+        native_value=lambda device: device.power / 1000 / device.voltage
+        if device.voltage
```

Comment:
we need to check both properties to be available
```suggestion
        if device.power and device.voltage
```